app/controllers/healthcheck_controller.rb

```diff
@@ -10,7 +10,10 @@ def check
 
   def overdue
     # Check the number of overdue editions
-    render json: { 'overdue' => Edition.due_for_publication.count }
+    render json: {
+      'overdue' => Edition.due_for_publication.count,
+      'missing_from_site' => [],
```

Comment:
Should this be a fetch of `MissingScheduledPublishing` instances that aren't `SUCCESSFULLY_PUBLISHED` ?